orchagent/qosorch.cpp

```diff
@@ -0,0 +1,11 @@
+#include "qosorch.h"
+#include "logger.h"
+
+namespace swss {
+
+void QosOrch::doTask(_in_ Consumer& consumer_info)
+{
+}
+
+}
+
```

Comment:
Can you split the qosorch to a different pull request or at least different commit?